### PR TITLE
Add gRPC example to DocService documentation

### PR DESCRIPTION
**Pull request by trustin**
Motivation:

It is not clear from `server-docservice.rst` that `DocService` supports
gRPC services or not.

Modifications:

- Update the code examples so that it is clear that both Thrift and gRPC
  services are supported

Result:

- gRPC users are happy.

### CODE REVIEW

File: site/src/sphinx/server-docservice.rst [hunk 1]
```diff
@@ -15,10 +15,21 @@ First, add :api:`DocService` to the :api:`ServerBuilder`:
...
     ServerBuilder sb = new ServerBuilder();
     sb.http(8080);
-    // Add some RPC services.
-    sb.service("/hello", THttpService.of(new MyHelloService());
-    // Add DocService.
+
+    // Add a Thrift service which implements 'ThriftHelloService'.
+    sb.service("/hello", THttpService.of(new MyThriftHelloService());
+
+    // Add a gRPC service which implemnets 'GrpcHelloService'.
```
Comment (hyangtack) [inline site/src/sphinx/server-docservice.rst]: `implemnets` -> `implements`

File: site/src/sphinx/server-docservice.rst [hunk 2]
```diff
@@ -15,10 +15,21 @@ First, add :api:`DocService` to the :api:`ServerBuilder`:
 
     ServerBuilder sb = new ServerBuilder();
     sb.http(8080);
-    // Add some RPC services.
-    sb.service("/hello", THttpService.of(new MyHelloService());
-    // Add DocService.
+
+    // Add a Thrift service which implements 'ThriftHelloService'.
+    sb.service("/hello", THttpService.of(new MyThriftHelloService());
```
Comment (minwoox) [inline site/src/sphinx/server-docservice.rst]: `())` -> `()))`

File: site/src/sphinx/server-docservice.rst [hunk 3]
```diff
@@ -89,8 +100,14 @@ when building a :api:`DocService` with a :api:`DocServiceBuilder`:
...
     sb.serviceUnder("/docs", new DocServiceBuilder()
+            // HTTP headers for all services
             .exampleHttpHeaders(HttpHeaders.of(AUTHORIZATION, "bearer b03c4fed1a"))
-            .exampleRequest(new HelloService.hello_args("Armeria"))
+            // Thrift example request for 'ThriftHelloService.hello()'
+            .exampleRequest(new ThriftHelloService.hello_args("Armeria"))
+            // gRPC example request for 'GrpcHelloService.Hello()'
+            .exampleRequestForMethod(GrpcHelloServiceGrpc.SERVICE_NAME,
+                                     "Hello", // Method name
+                                     HelloRequest.newBuilder().setName("Armeria").build());
```
Comment (minwoox) [inline site/src/sphinx/server-docservice.rst]: `build());` -> `build())`

**Comment by trustin** [comment]
Thanks. Fixed all. Will merge once CI passes.

**Pull request merged**